scripts/main/contextMenu.js

```diff
@@ -110,21 +110,21 @@ contextMenu.album = function (albumID, e) {
 	// Show merge-item when there's more than one album
 	// Commented out because it doesn't consider subalbums or shared albums.
 	// let showMerge = (albums.json && albums.json.albums && Object.keys(albums.json.albums).length>1);
-	const showMerge = true;
+	const showMergeMove = !albums.isTagAlbum(albumID);
```

Comment:
I assume the comment above this line needs some updating, too.

Reply:
Honestly, I'm inclined to simply remove the commented out code plus its associated comments. That "feature" is never coming back...

Reply:
Fine for me. Better no comment than a comment which raises more questions than it answers.